client-js/app/elements/labrad-registry.html

```diff
@@ -94,11 +99,16 @@
     <div id="reg-area">
       <table>
         <tbody is="selectable-table" attr-for-selected="name" selected="{{selDir}}" on-tap="selectDir">
-          <template id="dirList" is="dom-repeat" items="{{dirs}}" filter="filterFunc">
+          <template id="dirList" is="dom-repeat" items="{{dirs}}" filter="filterFunc"> 
             <tr name="{{item.name}}">
-              <td>
+              <td class="dir" 
+                  name="{{item.name}}" 
+                  draggable="true" 
+                  ondragover="classList.add('over')" 
+                  ondragleave="classList.remove('over')" 
```

Comment:
we should try to keep code separate from layout; can these be moved to the .ts file?